app.js

```diff
@@ -0,0 +1,35 @@
+const express = require("express")
+const app = express()
+const {getTopics} = require("./controllers/get.controllers")
+const e = require("express")
+
+app.use(express.json())
```

Comment:
Is this boiler plate for future use? You do not need to require this just yet.

Reply:
Yeah it is. I struggled on Thursday/Friday because I kept missing things so I did a boiler plate of things I'm likely to need.